apps/DemoApp/DemoAppUITests/DemoAppUITests.m

```diff
@@ -38,4 +38,32 @@ - (void)testPlayVideo {
     }
 }
 
+- (void)testIMASDK {
+    XCUIApplication *app = [[XCUIApplication alloc] init];
+    [app setLaunchEnvironment:@{@"ENV_KEY": @"tr4q3qahs0gflm8b1c75h49ln", @"TEST_SCENARIO": @"IMA"}];
```

Comment:
is this env key meaningful or random?

Reply:
It's the one used for the CI generated property

Reply:
I thought in CI we were pulling that from an env var?

Reply:
We haven't done that so far, but that is a good idea. What the tests do right now is pass in the property key in the launch environment to the app. But we could look into having this be an env var in BuildKite that gets passed into the tests instead